tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/language_script.tsv

```diff
@@ -272,6 +272,7 @@ gl	Galician	primary	Latn	Latin
 gld	Nanai	primary	Cyrl	Cyrillic
 glk	Gilaki	primary	Arab	Arabic
 gmh	Middle High German	secondary	Latn	Latin
+gmy	Mycenaean Greek	primary	Linb	Linear B
```

Comment:
in fact per wikipedia wouldn't it be `gmy_Cprt` and not `grc_Cprt` anyway?

Reply:
Mycenaean Greek (gmy) predates the Cypriot Syllabary -- it was written in Linear B.

Reply:
It's all… no, nevermind. I won't say it. Thx